source/windows/custom_build_dialog_window.py

```diff
@@ -105,6 +105,8 @@ def __init__(
                 str(file.relative_to(path)) for file in path.iterdir() if file.is_file() and os.access(file, os.X_OK)
             ]
         completer = QCompleter(executables, self)
+        completer.setCaseSensitivity(Qt.CaseSensitivity.CaseInsensitive)
```

Comment:
@Victor-IX Is this what you had in mind?

Reply:
Yep, this is perfect! Now Upper case or Lowercase doesn't matter